core/src/main/java/com/mapzen/android/graphics/model/BubbleWrapStyle.java

```diff
@@ -1,14 +1,52 @@
 package com.mapzen.android.graphics.model;
 
+import java.util.Set;
+
 /**
  * BubbleWrap default map style.
  */
-public class BubbleWrapStyle extends MapStyle {
+public class BubbleWrapStyle extends ThemedMapStyle {
 
   /**
    * Creates a new instance.
    */
   public BubbleWrapStyle() {
-    super("styles/bubble-wrap/bubble-wrap-style-more-labels.yaml");
+    super("styles/bubble-wrap/bubble-wrap-style.yaml");
+  }
+
+  @Override public String getBaseStyleFilename() {
+    return "bubble-wrap-style.yaml";
+  }
+
+  @Override public String getStyleRootPath() {
+    return "styles/bubble-wrap/";
+  }
+
+  @Override public String getThemesPath() {
+    return "themes/";
+  }
+
+  @Override public int getDefaultLod() {
+    return NONE;
+  }
+
+  @Override public int getLodCount() {
+    return NONE;
+  }
+
+  @Override public int getDefaultLabelLevel() {
+    return 5;
+  }
+
+  @Override public int getLabelCount() {
+    return 12;
```

Comment:
Is this 11 or 12?

Reply:
Its zero indexed and there are 12 label levels for bubble wrap: https://mapzen.com/documentation/cartography/themes/#bubble-wrap

Reply:
oops :D